src/components/OrdersWidget/OrderRow.tsx

```diff
@@ -151,19 +162,50 @@ const Status: React.FC<Pick<Props, 'order' | 'isOverBalance' | 'transactionHash'
   ])
   const isFilled = useMemo(() => isOrderFilled(order), [order])
   // Display isLowBalance warning only for active and partial fill orders
-  const isLowBalance = isOverBalance && !isUnlimited && !isFilled && !isScheduled && !isPendingOrder && !isExpiredOrder
+  const isLowBalance =
+    isOverBalance &&
+    !isUnlimited &&
+    !isFilled &&
+    !isScheduled &&
+    !isActiveNextBatch &&
+    !isPendingOrder &&
+    !isExpiredOrder
 
   const pending = useMemo(() => isPendingOrder && <PendingLink transactionHash={transactionHash} />, [
     isPendingOrder,
     transactionHash,
   ])
 
+  // Dima's trick to force component update
+  const [, forceUpdate] = useSafeState({})
+
+  // Updates the component every second, if it has one of the states where it should happen
+  useEffect(() => {
+    let timer: null | NodeJS.Timeout = null
+
+    if (isActiveNextBatch || isFirstActiveBatch) {
+      timer = setInterval(() => forceUpdate({}), 1000)
+    } else if (!!timer) {
```

Comment:
This is always `false`, because this `timer` is always `L184: let timer`, never `L187: timer = ...` because of scope

Reply:
You are doing this to `clearTimeout(lnterval` :wink: ) after next active batch and first active batch has passed (both `isActiveNextBatch, isFirstActiveBatch` are false)? Then you need `timeout` in outer scope to keep it over rerenders, put it in `useRef`

Reply:
`(both isActiveNextBatch, isFirstActiveBatch are false)? ` <- yes!

`useRef`, will do!